node-tests/e2e/fastboot-support-test.js

```diff
@@ -110,6 +110,30 @@ describe('e2e: fastboot integration', function () {
     });
   });
 
+  describe('scenario: with reportOnly = false', function () {
+    before(async function () {
+      await setConfig(testProject, { reportOnly: false });
+      await testProject.runEmberCommand('build');
+      await startServer();
+    });
+
+    after(async function () {
+      await stopServer();
+      await removeConfig(testProject);
+    });
+
+    it.only('sets CSP header if served via FastBoot', async function () {
+      let response = await request({
+        url: 'http://localhost:49742',
+        headers: {
+          Accept: 'text/html',
+        },
+      });
+
+      expect(response.headers).to.include.key('content-security-policy');
```

Comment:
I had the time to debug why the test is passing on `master` without this change. It is setting the CSP header as expected even though it throws. This could be detected by asserting the HTTP status code as well. It is `500 Internal Server Error` if the assertion is thrown.

```suggestion
      expect(response.statusCode).to.equal(200);
      expect(response.headers).to.include.key('content-security-policy');
```

Would be great if you could add the same assertion before line 107. It is missing there as well.